From 6176d68fc8e9717d2c96ff55e29a42c48e1395e8 Mon Sep 17 00:00:00 2001
From: Nick Schermer <nick@xfce.org>
Date: Sun, 6 Jan 2013 12:20:20 +0100
Subject: [PATCH 049/219] Autotools updates.

---
 configure.ac.in | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/configure.ac b/configure.ac
index aca782a..633b340 100644
--- a/configure.ac
+++ b/configure.ac
@@ -9,8 +9,8 @@ m4_define([xfpm_version], [xfpm_version_major().xfpm_version_minor().xfpm_versio
 AC_INIT([xfce4-power-manager], [xfpm_version], [http://bugzilla.xfce.org/])
 AC_PREREQ(2.50)
 
-AM_INIT_AUTOMAKE([1.8 dist-bzip2 tar-ustar])
-AM_CONFIG_HEADER([config.h])
+AM_INIT_AUTOMAKE([1.8 dist-bzip2 tar-ustar no-dist-gzip])
+AC_CONFIG_HEADERS([config.h])
 AM_MAINTAINER_MODE()
 m4_ifdef([AM_SILENT_RULES], [AM_SILENT_RULES([yes])])
 
-- 
1.9.3

