From 3387f3fe0abfb454b5d3596203a790048cd8f943 Mon Sep 17 00:00:00 2001
From: Eric Koegel <eric.koegel@gmail.com>
Date: Mon, 28 Apr 2014 18:32:48 +0300
Subject: [PATCH 202/219] Uninitialized variable: start_time

---
 src/xfpm-polkit.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/xfpm-polkit.c b/src/xfpm-polkit.c
index ccc4233..221c2d8 100644
--- a/src/xfpm-polkit.c
+++ b/src/xfpm-polkit.c
@@ -116,7 +116,7 @@ get_kinfo_proc (pid_t pid, struct kinfo_proc *p)
 static guint64
 get_start_time_for_pid (pid_t pid)
 {
-    guint64 start_time;
+    guint64 start_time = 0;
 #if defined(__linux)
     gchar *filename;
     gchar *contents;
-- 
1.9.3

