--- xfce4-power-manager-0.8.5/settings/Makefile.am~	2010-03-02 16:17:02.000000000 +0100
+++ xfce4-power-manager-0.8.5/settings/Makefile.am	2010-11-12 00:17:07.569248500 +0100
@@ -28,6 +28,7 @@
 	$(XFCONF_CFLAGS)
 
 xfce4_power_manager_settings_LDADD   =				\
+	$(top_builddir)/libxfpm/libxfpmcommon.la		\
 	$(GTK_LIBS)						\
 	$(GLIB_LIBS)						\
 	$(GTHREAD_LIBS)						\
@@ -36,8 +37,7 @@
 	$(DBUS_GLIB_LIBS)					\
 	$(LIBXFCE4GUI_LIBS)					\
 	$(LIBXFCE4UTIL_LIBS)					\
-	$(XFCONF_LIBS)						\
-	$(top_builddir)/libxfpm/libxfpmcommon.la
+	$(XFCONF_LIBS)
 
 
 manpagedir = $(mandir)/man1
