diff -urN xfce4-power-manager-1.0.3/configure.ac xfce4-power-manager-1.0.3.new//configure.ac
--- xfce4-power-manager-1.0.3/configure.ac	2011-01-04 13:37:58.000000000 +0000
+++ xfce4-power-manager-1.0.3.new//configure.ac	2011-02-11 19:45:25.683334784 +0000
@@ -251,7 +251,7 @@
 	======================================
  
 	prefix:			${prefix}
-	xdg autostart:		${sysconfdir/xdg/autostart}
+	xdg autostart:		${sysconfdir}/xdg/autostart
 	POLKIT:			${polkit}
    	DPMS:   		${have_dpms}
 	Network manager:	${ac_network_manager}
diff -urN xfce4-power-manager-1.0.3/src/Makefile.am xfce4-power-manager-1.0.3.new//src/Makefile.am
--- xfce4-power-manager-1.0.3/src/Makefile.am	2011-01-04 12:22:56.000000000 +0000
+++ xfce4-power-manager-1.0.3.new//src/Makefile.am	2011-02-11 19:45:51.306668083 +0000
@@ -97,10 +97,11 @@
 	$(PLATFORM_LDFLAGS)
 
 xfce4_power_information_LDADD =			\
+	$(top_builddir)/libdbus/libxfpmdbus.la	\
 	$(GOBJECT_LIBS)                         \
 	$(DBUS_GLIB_LIBS)                       \
-	$(LIBXFCE4UI_LIBS)			\
-	$(top_builddir)/libdbus/libxfpmdbus.la
+	$(LIBXFCE4UI_LIBS)
+
 
 if ENABLE_POLKIT
 
