--- xfce4-power-manager-1.2.0/src/xfpm-enum-types.c.orig	2014-05-10 20:06:28.630845667 +0200
+++ xfce4-power-manager-1.2.0/src/xfpm-enum-types.c	2014-05-10 20:06:39.554119824 +0200
@@ -7,46 +7,6 @@
 
 /* enumerations from "./xfpm-enum-glib.h" */
 GType
-xfpm_device_type_get_type (void)
-{
-	static GType type = 0;
-	if (type == 0) {
-	static const GEnumValue values[] = {
-	{ XFPM_DEVICE_TYPE_UNKNOWN, "XFPM_DEVICE_TYPE_UNKNOWN", "unknown" },
-	{ XFPM_DEVICE_TYPE_LINE_POWER, "XFPM_DEVICE_TYPE_LINE_POWER", "line-power" },
-	{ XFPM_DEVICE_TYPE_BATTERY, "XFPM_DEVICE_TYPE_BATTERY", "battery" },
-	{ XFPM_DEVICE_TYPE_UPS, "XFPM_DEVICE_TYPE_UPS", "ups" },
-	{ XFPM_DEVICE_TYPE_MONITOR, "XFPM_DEVICE_TYPE_MONITOR", "monitor" },
-	{ XFPM_DEVICE_TYPE_MOUSE, "XFPM_DEVICE_TYPE_MOUSE", "mouse" },
-	{ XFPM_DEVICE_TYPE_KBD, "XFPM_DEVICE_TYPE_KBD", "kbd" },
-	{ XFPM_DEVICE_TYPE_PDA, "XFPM_DEVICE_TYPE_PDA", "pda" },
-	{ XFPM_DEVICE_TYPE_PHONE, "XFPM_DEVICE_TYPE_PHONE", "phone" },
-	{ 0, NULL, NULL }
-	};
-	type = g_enum_register_static ("XfpmDeviceType", values);
-  }
-	return type;
-}
-GType
-xfpm_device_state_get_type (void)
-{
-	static GType type = 0;
-	if (type == 0) {
-	static const GEnumValue values[] = {
-	{ XFPM_DEVICE_STATE_UNKNOWN, "XFPM_DEVICE_STATE_UNKNOWN", "unknown" },
-	{ XFPM_DEVICE_STATE_CHARGING, "XFPM_DEVICE_STATE_CHARGING", "charging" },
-	{ XFPM_DEVICE_STATE_DISCHARGING, "XFPM_DEVICE_STATE_DISCHARGING", "discharging" },
-	{ XFPM_DEVICE_STATE_EMPTY, "XFPM_DEVICE_STATE_EMPTY", "empty" },
-	{ XFPM_DEVICE_STATE_FULLY_CHARGED, "XFPM_DEVICE_STATE_FULLY_CHARGED", "fully-charged" },
-	{ XFPM_DEVICE_STATE_PENDING_CHARGING, "XFPM_DEVICE_STATE_PENDING_CHARGING", "pending-charging" },
-	{ XFPM_DEVICE_STATE_PENDING_DISCHARGING, "XFPM_DEVICE_STATE_PENDING_DISCHARGING", "pending-discharging" },
-	{ 0, NULL, NULL }
-	};
-	type = g_enum_register_static ("XfpmDeviceState", values);
-  }
-	return type;
-}
-GType
 xfpm_battery_charge_get_type (void)
 {
 	static GType type = 0;
@@ -109,6 +69,8 @@
 	{ BUTTON_MON_BRIGHTNESS_DOWN, "BUTTON_MON_BRIGHTNESS_DOWN", "button-mon-brightness-down" },
 	{ BUTTON_LID_CLOSED, "BUTTON_LID_CLOSED", "button-lid-closed" },
 	{ BUTTON_BATTERY, "BUTTON_BATTERY", "button-battery" },
+	{ BUTTON_KBD_BRIGHTNESS_UP, "BUTTON_KBD_BRIGHTNESS_UP", "button-kbd-brightness-up" },
+	{ BUTTON_KBD_BRIGHTNESS_DOWN, "BUTTON_KBD_BRIGHTNESS_DOWN", "button-kbd-brightness-down" },
 	{ NUMBER_OF_BUTTONS, "NUMBER_OF_BUTTONS", "number-of-buttons" },
 	{ 0, NULL, NULL }
 	};
--- xfce4-power-manager-1.2.0/src/xfpm-enum-types.h.orig	2014-05-10 20:06:34.087482773 +0200
+++ xfce4-power-manager-1.2.0/src/xfpm-enum-types.h	2014-05-10 20:06:39.437453789 +0200
@@ -6,10 +6,6 @@
 #include <glib-object.h>
 G_BEGIN_DECLS
 /* enumerations from "./xfpm-enum-glib.h" */
-GType xfpm_device_type_get_type (void);
-#define XFPM_TYPE_DEVICE_TYPE (xfpm_device_type_get_type())
-GType xfpm_device_state_get_type (void);
-#define XFPM_TYPE_DEVICE_STATE (xfpm_device_state_get_type())
 GType xfpm_battery_charge_get_type (void);
 #define XFPM_TYPE_BATTERY_CHARGE (xfpm_battery_charge_get_type())
 GType xfpm_shutdown_request_get_type (void);
